core/src/sumeragi/main_loop.rs

```diff
@@ -650,6 +650,7 @@ fn should_terminate(shutdown_receiver: &mut tokio::sync::oneshot::Receiver<()>)
     }
 }
 
+#[instrument(skip_all)]
 /// Execute the main loop of [`SumeragiWithFault`]
 #[instrument(skip(sumeragi, state, genesis_network, shutdown_receiver))]
```

Comment:
Shouldn't we remove this attribute then?

Reply:
Conflict artefact.